Tests/IntegrationTests/SwiftKafkaTests.swift

```diff
@@ -489,4 +494,115 @@ final class SwiftKafkaTests: XCTestCase {
             XCTAssertTrue(acknowledgedMessages.contains(where: { $0.value == ByteBuffer(string: message.value) }))
         }
     }
+
+    func testProduceAndConsumeWithTransaction() async throws {
```

Comment:
This is just an example test so far to demonstrate how current API would look like from user perpective